src/Browser.ts

```diff
@@ -501,13 +495,13 @@ export class Browser {
 
         const res = await got<T>(this.#forceAbsoluteURL(url), options);
 
-        if (res.statusCode >= 400) {
+        if (res?.statusCode >= 400) {
             throw new ChatExchangeError(
-                `Remote server threw ${res.statusCode} error.`
+                `Remote server threw ${res.statusCode} error\nURL: ${url}`
             );
         }
 
-        return res;
+        return res||{body: { url: "" }};
```

Comment:
Fix the spacing:

`res || {body: { url: "" }}`

Double check again `res` though, because the `||` may not be necessary.

Reply:
Both things with guards are on me, sorry :) They should not be here